dapps/evancore.vue.libs/src/components/login/login.vue

```diff
@@ -18,46 +18,54 @@
 */
 
 <template>
-  <div class="bg-level-3 w-100 h-100 d-flex align-items-center flex-column">
+  <div class="bg-level-3 w-100 h-100 d-flex align-items-center justify-content-center flex-column">
     <div class="mt-3 mb-3 text-center">
-      <br>
-      <h1 class="mt-4 font-weight-semibold">{{ '_evan.welcome-to-evan' | translate }}</h1>
-      <h3 class="mt-4 font-weight-semibold text-muted">{{ '_evan.please-login' | translate }}</h3>
+      <h1>{{ '_evan.log-in' | translate }}</h1>
       <div class="bg-primary d-inline-block" style="width: 70px; height: 5px;"></div>
-      <br>
     </div>
     <div class="bg-level-1 mx-auto border password-dialog mt-3 mt-md-5">
-      <div class="d-flex p-2 align-items-center justify-content-between border-bottom">
-        <h4 class="m-0 ml-3">{{ '_evan.login' | translate }}</h4>
-        <evan-logout ref="evanLogout">
-          <template v-slot:button>
-            <button type="button" class="btn"
-              @click="$refs.evanLogout.logout()">
-              <i class="mdi mdi-logout"></i>
-            </button>
-          </template>
-        </evan-logout>
-      </div>
       <form class="p-4" v-on:submit.prevent="login">
         <div class="form-group">
+          <template v-if="alias">
+            <label for="alias">{{ '_evan.alias' | translate }}</label>
+            <input class="form-control" id="alias" type="text" :value="alias" disabled />
+
+            <evan-logout ref="evanLogout">
+              <template v-slot:button>
+                <div style="display:flex">
+                  <div class="spacer"></div>
```

Comment:
You can use the bootstrap styling: 

```suggestion
                  <div class="flex-grow-1"></div>
```